packages/cli/src/commands/epochs/switch.ts

```diff
@@ -0,0 +1,30 @@
+import { BaseCommand } from '../../base'
+import { displaySendTx } from '../../utils/cli'
+import { CustomFlags } from '../../utils/command'
+
+export default class Switch extends BaseCommand {
+  static description = 'Finishes current epoch and starts a new one.'
+
+  static flags = {
+    ...BaseCommand.flags,
+    // TODO not sure if this flag is needed at all here
+    from: CustomFlags.address({ required: true }),
+  }
+
+  static args = {}
+
+  static examples = ['switch --from 0x47e172F6CfB6c7D01C1574fa3E2Be7CC73269D95']
+
+  async run() {
+    const kit = await this.getKit()
+    const res = await this.parse(Switch)
+    const address = res.flags.from
+
+    kit.defaultAccount = address
+
+    const epochManager = await kit.contracts.getEpochManager()
+
+    await displaySendTx('finishNextEpoch', await epochManager.prepareFinishNextEpochProcessTx())
```

Comment:
wouldnt you start before you finish?

Reply:
this is for switching epochs meaning one is currently running, you finish it and start a new one

Reply:
I guess it's just semantics at this point, if it's called `finish` then why does it start a new one etc, we could call it `finish-and-start` to be 100% explicit, but the idea is that we want ideally to finish and start a new one in as little blocks as possible, so there's no gap in between

Reply:
No, this is wrong.

To go to the next epoch, is a 2 steps process..

 1. `startNextEpochProcess`
 2. `finishNextEpochProcess`

the start and finish refer to the 'process of changing the epoch' not to the epoch themselves.. so it's 
starting process, finishing process.

so.

call needs to be inverted!

Reply:
that makes way more sense!